__tests__/server.test.js

```diff
@@ -80,16 +79,65 @@ describe("GET /api/articles", () => {
         expect(articles.length).toBe(13);
 
         articles.forEach((article) => {
-          expect(typeof article.author).toBe("string");
-          expect(typeof article.title).toBe("string");
-          expect(Number.isFinite(article.article_id)).toBe(true);
-          expect(Number.isFinite(article.votes)).toBe(true);
-          expect(typeof article.topic).toBe("string");
-          expect(typeof article.created_at).toBe("string");
-          expect(typeof article.article_img_url).toBe("string");
+          expect(article).not.toHaveProperty("body");
+          expect(article).toHaveProperty("comment_count");
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            article_img_url: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            comment_count: expect.any(String),
+          });
+        });
+
+        const sortedArticles = [...articles].sort(
+          (a, b) => a.created_at - b.created_at
+        );
+        expect(articles).toEqual(sortedArticles);
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comment objects for the given article_id", () => {
+    return request(server)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments.length).toBe(2);
+
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
We can be a bit more specific here - all comments for this specific article should have an article_id of 3 attached to them :)